app/models/service.rb

```diff
@@ -30,9 +30,11 @@ class Service < ActiveRecord::Base
       add_attribute :_geoloc do
         if addresses.any?
           addresses.map do |a|
-            { lat: a.address_latitude, lng: a.address_longitude }
+            if a.address_latitude.present? & a.address_longitude.present?
```

Comment:
In Ruby, the single `&` operator is the bit-wise AND operator. You really should be using `&&`.

Reply:
Also, Rubocop doesn't like this style. Can you please fix it so that people branching from master don't unconditionally fail Rubocop? In general, we should at least wait for CI to pass before merging, unless if we have a strong reason to not wait.